openwisp_notifications/tests/test_notifications.py

```diff
@@ -953,37 +953,50 @@ def test_batch_email_notification_with_descriptions(self, mock_send_email):
         self.assertEqual(len(mail.outbox), 1)
 
         # Call the task
-        tasks.batch_email_notification(self.admin.id)
+        tasks.send_batched_email_notifications(self.admin.id)
 
         # Check if the rest of the notifications are sent in a batch
         self.assertEqual(len(mail.outbox), 2)
-        self.assertIn('Summary of 4 Notifications', mail.outbox[1].subject)
+        self.assertIn('4 new notifications since', mail.outbox[1].subject)
         self.assertNotIn('View all Notifications', mail.outbox[1].body)
         self.assertIn('Test Notification', mail.outbox[1].body)
 
-    @patch('openwisp_notifications.tasks.batch_email_notification.apply_async')
+    @patch('openwisp_notifications.tasks.send_batched_email_notifications.apply_async')
     def test_batch_email_notification_with_call_to_action(self, mock_send_email):
         self.notification_options.update(
             {
                 'message': 'Notification title',
                 'type': 'default',
             }
         )
-        for _ in range(11):
+        for _ in range(21):
             notify.send(recipient=self.admin, **self.notification_options)
 
         # Check if only one mail is sent initially
         self.assertEqual(len(mail.outbox), 1)
 
         # Call the task
-        tasks.batch_email_notification(self.admin.id)
+        tasks.send_batched_email_notifications(self.admin.id)
 
         # Check if the rest of the notifications are sent in a batch
         self.assertEqual(len(mail.outbox), 2)
-        self.assertIn('Summary of 10 Notifications', mail.outbox[1].subject)
+        self.assertIn('20 new notifications since', mail.outbox[1].subject)
         self.assertIn('View all Notifications', mail.outbox[1].body)
         self.assertNotIn('Test Notification', mail.outbox[1].body)
 
+    @override_settings(EMAIL_BATCH_INTERVAL=0)
+    def test_without_batch_email_notification(self):
+        self.notification_options.update(
+            {
+                'message': 'Notification title',
+                'type': 'default',
+            }
+        )
+        for _ in range(3):
+            notify.send(recipient=self.admin, **self.notification_options)
+
+        self.assertEqual(len(mail.outbox), 3)
```

Comment:
So, is this working now @Dhanus3133?

Reply:
No, it's not. I'm not sure why CI for the PR isn't running.

Reply:
Seems Gagan fixed it. It seems it will be triggered at the next commit.